contracts/fixed_price_rev1.1.scilla

```diff
@@ -131,7 +144,13 @@ let make_error =
 
 contract FixedPriceOrderbook
 (
-  initial_contract_owner: ByStr20
+  initial_contract_owner: ByStr20,
+  collection_contract: ByStr20 with contract
```

Comment:
@sam-zilliqa can this be set to zero address for other applications which do not intend to use a brand collection? Just like allowlist_address can be zero.

Reply:
Not possible at the moment (as discussed in the call earlier today)